pkg/cloud/ovirt/clients/machineservice.go

```diff
@@ -323,8 +323,51 @@ func (is *InstanceService) handleNics(vmService *ovirtsdk.VmService, spec *ovirt
 	return nil
 }
 
+func (is *InstanceService) getAffinityGroups(cID string, agNames []string) (ag []*ovirtsdk.AffinityGroup, err error) {
+	var ags []*ovirtsdk.AffinityGroup
+	res, err := is.Connection.SystemService().ClustersService().
+		ClusterService(cID).AffinityGroupsService().
+		List().Send()
+	if err != nil {
+		return nil, err
+	}
+	agNamesMap := make(map[string]*ovirtsdk.AffinityGroup)
+	for _, af := range res.MustGroups().Slice() {
+		agNamesMap[af.MustName()] = af
+	}
+	for _, agName := range agNames {
+		if _, ok := agNamesMap[agName]; ok {
+			ags = append(ags, agNamesMap[agName])
+		}
+	}
+	return ags, nil
+}
+
+func (is *InstanceService) handleAffinityGroups(vm *ovirtsdk.Vm, cID string, agsName []string) error {
```

Comment:
Where are these methods used? This seems to be new code but it isn't called anywhere?

Reply:
Wow really stupid mistake...
Lost the call in the rebase, I'll return it